core/include/dt-bindings/input/input.h

```diff
@@ -0,0 +1,18 @@
+/* SPDX-License-Identifier: GPL-2.0 */
```

Comment:
We may have a problem with this (no BSD-compatible license)

Reply:
Oh! I missed this one.
Until this get solved in a nice way and as stm32mp1 OP-TEE Core does not deal with input bindings, I think I'll modify the DTS file remove data related to these bindings.
Sound fair?

Reply:
Fine with me. Perhaps you could further minimize the changes by adding the 5 needed defines at the beginning of the file instead? (`KEY_ENTER` etc.). As you like.

Reply:
I guess that would not clearly tackle the license issue.
I would prefer to wait a cleaner way. The system device tree initiative (sharing DTS files among firmwares in a system) should address this in one way or another I guess.